README.md

```diff
@@ -4,7 +4,9 @@
 
 _Matrix Algebra proGrams In Common Lisp_ by [Rigetti Computing](http://www.rigetti.com). (née FLAIL: _Finally, Linear Algebra In Lisp!_)
 
-Load it with `(ql:quickload :magicl)`.
+To begin, clone this repository into `quicklisp/local-projects`.
```

Comment:
This could be clearer, IMO. Quicklisp may've been installed into a non-standard location. Say something like "into your `ql:*local-project-directories*`."